lib/connectors_sdk/confluence_cloud/http_call_wrapper.rb

```diff
@@ -17,12 +17,29 @@ module ConfluenceCloud
     class HttpCallWrapper < ConnectorsSdk::Base::HttpCallWrapper
       SERVICE_TYPE = 'confluence_cloud'
 
-      def name
+      def display_name
         'Confluence Cloud'
       end
 
-      def service_type
-        SERVICE_TYPE
+      def connection_requires_redirect
+        true
+      end
+
+      def configurable_fields
+        [
+          {
+            'key' => 'base_url',
```

Comment:
Do you think we should prefix config fields from external connector with `external_connector_`? e.g. `external_connector_base_url`

Reply:
Not a bad idea, although the value it brings is avoiding collision with other configuration fields, which happen to be `external_connector_url` and `external_connector_api_key`, which of course already contain the _external_connector__ prefix. What I _really_ wanted to do was introduce a new hash level for these fields, but that would've increased a lot of scope. We have a lot of code that assumes those values all live in a single, flat hash. At this point I suspect it's not worth the effort.